js/foam/ui/md/Halo.js

```diff
@@ -199,30 +203,35 @@ CLASS({
   listeners: [
     {
       name: 'onMouseDown',
-      code: function(evt) {;
+      code: function(evt) {
```

Comment:
this makes it that clicking on a element does not show a visible focus UI but still switches the focus to the clicked element for future keyboard interactions.

Reply:
Arrow function?

Reply:
Do we want to deviate from the style of the rest of the codebase? I'm happy to use arrow functions but thought i rather as well match the way the current codebase works

Reply:
I'll leave it to you to decide, but my preference is generally to use an arrow function and go for incremental improvements. I can be convinced by arguments of consistency but I have a preference for making code "better" (for subjective definition better) if possible.

Reply:
Fair nuff, will update!

Reply:
turns out these functions get .bind called on them via the magic of foam and as such doing arrow functions breaks them. Back to anon functions we go!